sphinx/cmd/quickstart.py

```diff
@@ -175,10 +175,7 @@ def _has_custom_template(self, template_name: str) -> bool:
               It will be removed in the future without deprecation period.
         """
         template = path.join(self.templatedir, path.basename(template_name))
-        if self.templatedir and path.exists(template):
-            return True
-        else:
-            return False
+        return bool(self.templatedir) and path.exists(template)
```

Comment:
Why do you call `bool()` here?

Reply:
mypy returns a false positive here without explicitly coercing to a bool. there might be a better way to address this. will investigate